specification/sql/resource-manager/Microsoft.Sql/preview/2024-08-01-preview/DatabaseAdvisors.json

```diff
@@ -406,7 +592,9 @@
         "details": {
           "description": "Gets additional details specific to this recommended action.",
           "type": "object",
-          "additionalProperties": true,
+          "additionalProperties": {
+            "type": "string"
+          },
```

Comment:
Use of additionalProperties is not allowed for properties owned by the service. The only time its ok to use it is when the properties are pass thru (user defined) and not subject to any validations. Please make this an array or provide an explanation for why you need this.

Reply:
This has been used since the inception of the service, additionalProperties is already being used and has been for the last 9 years, I just wanted to fix the ModelValidation error, as the SQL ARM API team asked me to do that.